packages/web-components/src/components/va-select/va-select.tsx

```diff
@@ -48,15 +48,14 @@ export class VaSelect {
    */
   @Prop() enableAnalytics: boolean;
 
-  /**
-   * The event attached to select's onkeydown
-   */
-  @Event() vaKeyDown: EventEmitter;
-
   /**
    * The event emitted when the selected value changes
    */
-  @Event() vaSelect: EventEmitter;
+  @Event({
+    bubbles: true,
+    composed: true,
+  })
+  vaSelect: EventEmitter;
```

Comment:
Oh, this is what should make that test easier for the app team, right? I was too focused on the `keydown` changes to notice this the first time :upside_down_face:

Reply:
The tests are still failing locally when using ```userEvent.click(await screen.findByText(/use your current location/i));```. 😞